### PR TITLE
#163589492 Display Average Rating Value on Articles

### PR DESCRIPTION
**Pull request by tejiri4**
#### What does this PR do?
This PR allow users to be able to see the average rating of an article when the article is displayed
#### Description of Task to be completed?
- update rating controller
- update article migration
- update article model

#### How should this be manually tested?
- Git clone the repo
- Cd into it
- Run npm install
- Run npm test
#### What are the relevant pivotal tracker stories?
[Average Rating](https://www.pivotaltracker.com/story/show/163589492)

### CODE REVIEW

File: server/test/controllers/comment.spec.js [hunk 1]
```diff
@@ -1,252 +1,10 @@
...
-
-
 import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import sinon from 'sinon';
 import app from '../../..';
 import db from '../../models';
 import { token, tokenTwo } from '../mockData/token';
-
+console.log(tokenTwo);
```
Comment (rafmme) [inline server/test/controllers/comment.spec.js]: Tejiri, please always ensure that you don't forget to remove console.log when you are through with debugging, thanks

Comment (tejiri4) [reply server/test/controllers/comment.spec.js]: Fixed that already thanks

**Review by rafmme — COMMENTED** [review]
LGTM, yet

File: server/controllers/RatingController.js [hunk 2]
```diff
@@ -118,9 +118,16 @@ class RatingController {
...
 
-      const rated = await Util.resourceExists(Rating, {
-        articleId: article.dataValues.id,
-        userId
+      const checkRating = await Rating.findAndCountAll({
+        where: {
+          articleId: article.dataValues.id
+        }
+      });
+      const { count } = checkRating;
```
Comment (Bobsar0) [inline server/controllers/RatingController.js]: Any reason for not destructuring rows as well?

File: server/helpers/Util.js [hunk 3]
```diff
@@ -83,6 +83,20 @@ class Util {
...
   }
 
+  /**
+   * @static
+   * @description a function to ascertain if a resource exists
+   * @param {object} model Model to check for resource existence
+   * @param {object} data
+   * @returns {object} returns resource if true or null otherwise
+   */
+  static async updateAverageRating(model, data) {
```
Comment (Bobsar0) [inline server/helpers/Util.js:93]: Nice helper function. However, please edit the JSdoc to suit the function

**Comment by Bobsar0** [comment]
LGTM, yet

**Pull request merged**